app/src/main/java/org/simple/clinic/bp/sync/BloodPressureSync.kt

```diff
@@ -8,6 +8,7 @@ import org.simple.clinic.bp.BloodPressureRepository
 import org.simple.clinic.sync.ModelSync
```

Comment:
`Move logged in user check out of 'DataSync'` -> `Move user authentication check out of DataSync`?

Reply:
Better. I've reworded the message.